test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -162,3 +166,31 @@ def test_tensorflow_addons_cpu(tensorflow_training, ec2_connection, cpu_only):
     if is_tf1(tensorflow_training):
         pytest.skip("This test is for TF2 only")
     execute_ec2_training_test(ec2_connection, tensorflow_training, TF_ADDONS_CMD)
+
+@pytest.mark.model("dataservice")
```

Comment:
is dataservice the model under test, or the integration? if integration:

```
@pytest.mark.integration('dataservice')
```

Model is a required marker, but. if this is not testing a model you can put "N/A" and maybe explain why in the function docstring